src/Packages/RetroMole.Render.Veldrid/Controller.cs

```diff
@@ -304,10 +296,10 @@ public override void RenderImDrawData(params object[] args)
                     else
                     {
                         if (pcmd.TextureId != IntPtr.Zero)

```

Comment:
This check currently never passes which seems to lead to nothing except a cleared texture to be sent to the gd

Reply:
The check was useless afaict so I just removed it